ulib/FStar.Reflection.Data.fsti

```diff
@@ -29,11 +29,13 @@ type vconst =
   | C_Reflect   : name -> vconst
   (* TODO: complete *)
 
+type universes = list universe
+
 // This is shadowing `pattern` from Prims (for smt_pats)
 noeq
 type pattern =
     | Pat_Constant : vconst -> pattern              // A built-in constant
-    | Pat_Cons     : fv -> list (pattern * bool) -> pattern
+    | Pat_Cons     : fv -> option (list universe) -> list (pattern * bool) -> pattern
```

Comment:
Minor: did you mean to use `universes` (its definition moved up).

Reply:
thanks